app/components/ProductNavigation.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+
+import {GridList, GridTile} from 'material-ui/GridList'
+import Paper from 'material-ui/Paper';
+
+
+export default () => {
+
+const styles = {
+  root: {
+    display: 'flex'
+  },
+  gridList: {
+    display: 'flex',
+    width: '100%',
+    alignItems: 'center',
+    justifyContent: 'center',
+    cellHeight: 'auto',
+    margin: 'auto'
+  },
+  gridTile: {// whyyyy height whyyyy
+      width: 'auto',
+      height: '100%',
+      margin: '5%'
+  },
+  titleStyle: {
+    color: '#FAFAFA',
+    fontSize: '1.2em'
+  },
+  paperStyle: {
+  margin: 'auto'
+}
+}
+
+const tilesData = [
+  {
+    img: 'https://scontent-lga3-1.xx.fbcdn.net/v/t1.0-9/12108231_10156342670650105_8218991482164709564_n.jpg?oh=67f9780ae4aace86d4891616fb84af6e&oe=58DF96BA',
```

Comment:
1. Can these images be stored locally? 
2. Can this array of data live on props instead of being hard-coded in? (This might be a rare exception in which you DO actually want to hard code this stuff in, just something to think about)